File: Example4OC/Example4OC/Classes/Map/BVShareMapController.m
//
//  BVShareMapController.m
//  LibosExample
//
//  Created by 黑眸智能 on 2021/11/2.
//

#import "BVShareMapController.h"

@interface BVShareMapController ()

@end

@implementation BVShareMapController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (NSString *)getNavTitle {
    return @"分享地图模式";
}

- (BVSweeperMapViewMode)getMapMode {
    return BVSweeperMapViewModeShareMode;
}

@end

File: Example4Swift/Example4Swift/Classes/Map/BVShareMapController.swift
//
//  BVShareMapController.swift
//  Libos
//
//  Created by 黑眸智能 on 2021/11/2.
//

import UIKit
import BVSweeperKit

class BVShareMapController: BVMapController {

    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func getNavTitle() -> String {
        "分享地图模式"
    }
    
    override func getMapMode() -> BVSweeperMapViewMode {
        .shareMode
    }
}
